packages/vaadin-tabs/src/vaadin-tab.js

```diff
@@ -4,9 +4,9 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ItemMixin } from '@vaadin/vaadin-item/src/vaadin-item-mixin.js';
```

Comment:
**nit:** Sort the mixins in alphabetical order.

Reply:
Done.